pallets/msa/src/lib.rs

```diff
@@ -953,31 +943,19 @@ pub mod pallet {
 		///
 		/// # Errors
 		/// - [`NoKeyExists`](Error::NoKeyExists) - If there is not MSA for `origin`.
+		/// - [`Error::ExceedsMaxProviderNameSize`] - Too long of a provider name
 		#[pallet::call_index(11)]
-		#[pallet::weight(T::WeightInfo::propose_to_be_provider())]
+		#[pallet::weight(T::WeightInfo::propose_to_be_provider_v2())]
```

Comment:
```suggestion
		#[pallet::weight(T::WeightInfo::propose_to_be_provider())]
```
I think this is the original, not the v2, correct?

Reply:
Or are we just using the same weights for both?

Reply:
I assume we can since its calling underlying extrinsic (re-routing after size check), you suggest we have separate benchmarks for both?